### PR TITLE
Add styles for the registration page

### PR DESCRIPTION
**Pull request by BohdanDudkaTD**
This PR adds style implementation for the `Registration page`.
The final result looks next:
image

Result of the ESLint execution:
image

### CODE REVIEW

**Comment by BohdanDudkaTD** [comment]
@dkalinin, PTAL.

**Comment by dkalinin** [comment]
Please add Oleh as a reviewer.

**Comment by BohdanDudkaTD** [comment]
@Oleh-Stefanovych, PTAL.

File: app/style/css/style.css [hunk 1]
```diff
@@ -45,6 +45,15 @@ hr {
     transition: 0.4s;
 }
 
+.submit-button.disabled {
+    background-color: #9cd9ff;
+}
+
+.submit-button.disabled:hover {
+    color: #fff;
+    background-color: #9cd9ff;
```
Comment (Oleh-Stefanovych) [inline app/style/css/style.css]: You already have this style in `.submit-button.disabled`

File: app/view/component/button/index.js [hunk 2]
```diff
@@ -63,11 +66,15 @@ export class Button {
...
+   * * @param {...string} classes - an array of strings which are names of classes of the button.
    * @private
    */
-  _render(parent, textContent) {
+  _render(parent, textContent, ...classes) {
     const button = document.createElement('button');
     button.textContent = textContent;
+    classes.forEach((className) => {
+      button.classList.add(className);
+    });
```
Comment (Oleh-Stefanovych) [inline app/view/component/button/index.js]: `button.classList.add(...classes);`

File: app/view/component/input/index.js [hunk 3]
```diff
@@ -130,13 +154,44 @@ export class Input {
...
+
+  /**
+   * Updates state of the span with error. If span is already rendered and
+   * error message is empty - remove the span. If error message is not empty - update span
+   * with new error message. If span is not rendered - render it to the parent element.
+   *
+   * @private
+   */
+  _updateErrorSpan() {
+    const errorSpan = this._parentDiv.querySelector('.errorSpan');
```
Comment (Oleh-Stefanovych) [inline app/view/component/input/index.js]: Let's have error span in `field` component.

**Comment by BohdanDudkaTD** [comment]
@Oleh-Stefanovych, PTAL.
Fixed issues:
- remove unnecessary `background-color` from the submit button styles;
- use spread syntax to add classes for the button element;
- move the logic of error span rendering from the `Input` component to the `Field`.

**Comment by BohdanDudkaTD** [comment]
@dkalinin, PTAL.
Fixed issues:
- fix issue when error message from the server was shown out of the box with registration form;
- add `...acceptableFiles` for the `Field` and `Input` components;
- don't clean `inputs with passwords` after their validation.

File: app/view/component/avatar-chooser/index.js [hunk 4]
```diff
@@ -43,14 +39,15 @@ export class AvatarChooser {
...
+    this._root.innerHTML = `
+      <div class="avatar-floater">
+        <img class="avatar-image" alt="User avatar" src="${this._defaultAvatarURL}"
+      </div>`;
+
+    const image = this._root.querySelector('.avatar-image');
 
     this._avatarField = new Field(this._root, this._inputId, 'Select your avatar',
-        'Select avatar', 'file');
+        'Select avatar', 'file', 'image/x-png', 'image/jpeg');
```
Comment (dkalinin) [inline app/view/component/avatar-chooser/index.js]: Validation accepts jpeg only:
image

**Pull request merged**